subsys/bluetooth/controller/Kconfig

```diff
@@ -7,6 +7,56 @@
 
 comment "BLE Controller support"
 
+# The following symbols are enabled depending if the controller actually
+# supports the respective features. In case of Nordic SW LL, all of these are
+# supported; for OpenISA SW LL, they're not
+
+config BT_LL_SW_LEGACY_SUPPORT
```

Comment:
```BT_LL_SW_LEGACY_SUPPORT``` is only used in one occurence below. It would remove it and change 
```depends on BT_LL_SW_LEGACY_SUPPORT -> depends on SOC_COMPATIBLE_NRF```

Reply:
Right, since it's the only controller that supports the legacy mode, and this will be deprecated. So there's a really low probability another SW LL implementation will show up, supporting the legacy mode. As such, I will do the change.